app/tests/integration/nimbus/pages/home.py

```diff
@@ -6,15 +6,14 @@
 class HomePage(Page):
     """Nimbus Home page."""
 
+    _page_wait_locator = (By.CSS_SELECTOR, "#PageHome-page")
```

Comment:
Interesting. This moved+changed. Why wait for `#pageHome-page` instead of `.directory-table`?

Reply:
1) Consistency, looking for the named outer container on every page makes it easier to reason about what we're waiting for
2) This is actually a bug in the test setup, `.directory-table` returns an element for each of the 6 tables, and if you just pass that into the wait function, it just picks the first one, which is fine if you land on the page with the Live tab highlighted, but if you set it to any of the other tabs, it picks the first one which is hidden, which fails the wait for check.